pg8000/tests/test_query.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 import threading
 import pg8000
-from .connection_settings import db_connect
+from pg8000.tests.connection_settings import db_connect
```

Comment:
This change doesn't really look correct.  Why was this changed away from a relative import?